nav.php

```diff
@@ -3,41 +3,52 @@
 if(session_id() == ''){session_start();}
 if(isset($_SESSION['login_user'])){
 ?>
-     
-<!-- This is for my burger navigation/menu with links to all my pages-->
-<nav>
-			<label>
-				<input type="checkbox">
-				<span class="menu"> <span class="hamburger"></span> </span>
+
+<!-- This is for my navigation menu with links to all my pages-->
+<nav class="navbar">
```

Comment:
#trialled using a navigation bar menu instead of burger menu that was used in version 1